src/tokens-studio/spectrum2-non-colors/spectrum2/layout.component/desktop.json

```diff
@@ -729,6 +729,16 @@
       }
     }
   },
+  "avatar-border-thickness": {
```

Comment:
@GarthDB I noticed this is the only component `borderWidth` token  type that is not called `...-border-width` in our system. And, it's not pointing to `border-width-100`. Can you help confirm this is correct?

Reply:
@GarthDB Sorry I missed your reply initially! 
Let's confirm with Kami on this – I left a Slack message for them, [here](https://adobedesign.slack.com/archives/C02LSCFPFR6/p1723062634819389?thread_ts=1718730302.013659&cid=C02LSCFPFR6). They're both out this week so I'm going to make the change in the data in the meantime.